spec/helpers/ajax-helper.ts

```diff
@@ -158,6 +158,14 @@ export class MockXMLHttpRequest {
     this.password = password;
     this.readyState = 1;
     this.triggerEvent('readyStateChange');
+    // https://developer.mozilla.org/en/docs/Web/API/XMLHttpRequest/Using_XMLHttpRequest
```

Comment:
just fyi, since `ajax-helper` is mock this comment may need to be placed under `AjaxObservable` implementation.